kodilla-advanced-tests/src/main/java/com/kodilla/execution_model/homework/Shop.java

```diff
@@ -9,34 +9,34 @@
 public class Shop {
     private final List<Order> orders = new ArrayList<>();
 
-    public List<Order> getOrder () {
+    public List<Order> getOrder() {
         return orders;
     }
-    public void addOrder (Order order){
+    public void addOrder(Order order) {
         this.orders.add(order);
     }
 
-    public List<Order> getOrderBtwTwoDates(LocalDate from, LocalDate to){
-        if(to.isBefore(from))
+    public List<Order> getOrderBtwTwoDates(LocalDate from, LocalDate to) {
+        if (to.isBefore(from))
             return Collections.emptyList();
         return orders.stream()
-                .filter(d -> d.getDate().isAfter(from)&& d.getDate().isBefore(to))
+                .filter(date -> date.getDate().isAfter(from)&& date.getDate().isBefore(to))
                 .collect(Collectors.toList());
     }
 
     public List<Order> getOrderFromValueRange(double min, double max) {
         if (orders.size() == 0)
             return Collections.emptyList();
         return orders.stream()
-                .filter(order -> order.getValue() >= min && order.getValue() <= max)
+                .filter(value -> value.getValue() >= min && value.getValue() <= max)
```

Comment:
czemu zamienilas na value? Streamujesz orders, czyli jeden to order - bylo dobrze. Value nic nie mowi, tym bardziej ze potem jest absurdalne value.getValue()